aries_vcx/src/libindy/proofs/proof_request.rs

```diff
@@ -1,290 +0,0 @@
-use std::collections::HashMap;
-use std::vec::Vec;
-
-use serde_json;
-
-use crate::error::prelude::*;
-use crate::libindy::proofs::proof_request_internal::{AttrInfo, NonRevokedInterval, PredicateInfo};
-use crate::libindy::utils::anoncreds;
-
-#[derive(Serialize, Deserialize, Builder, Debug, PartialEq, Clone)]
-#[builder(setter(into), default)]
-pub struct ProofRequestData {
```

Comment:
These are indy specific structures which I think should stay in `aries-vcx`. From messages crate perspective, we can work with this just as attachment. I think with few modifications to code, messages crate does not need to know this `ProofRequestData` exists